section_faiss_vector_index.go

```diff
@@ -763,3 +763,48 @@ func (v *vectorIndexOpaque) Reset() (err error) {
 
 func (v *vectorIndexOpaque) Set(key string, val interface{}) {
 }
+
+// -----------------------------------------------------------------------------
+
+// params:
+//   - SegmentBase
+//   - offset to the start of the vector section for the field
+func getVectorSectionContentOffsets(sb *SegmentBase, offset uint64) (
+	docValueStart uint64,
+	docValueEnd uint64,
+	indexBytesLen uint64,
+	indexBytesOffset uint64,
+	numVecs uint64,
+	vecDocIDsMappingOffset uint64,
```

Comment:
Refactor the order of the return values to reflect the order in which the content is retrieved.

Reply:
Done
Thanks